lib/Symbol.pm

```diff
@@ -132,6 +134,7 @@ sub qualify ($;$) {
 }
 
 sub qualify_to_ref ($;$) {
```

Comment:
maybe we could consider to use function signature instead of prototype? (as part of a next PR)

Reply:
That's nonsense. Signatures and prototypes are unrelated features.